client/signup/steps/theme-selection/style.scss

```diff
@@ -1,44 +1,8 @@
-.theme-thumbnail__name {
-	background-color: $gray-light;
-	box-shadow: 0 0 0 1px transparentize( lighten( $gray, 20% ), .5 ), 0 1px 2px lighten( $gray, 30% );
-	display: block;
-	font-size: 16px;
-	padding: 12px 0;
-	text-align: center;
+.themes-list .theme__active-focus {
+	opacity: 0.0;
```

Comment:
Third, any CSS change needs to be namespaced properly for the context it's being used on: this change is being applied everywhere! :)